release-notes.md

```diff
@@ -0,0 +1,106 @@
+[//]: # (<details>)
+
+[//]: # (<summary>Release Notes v1.0.1</summary>)
+
+[//]: # (<!-- always an empty line before table -->)
+
+## Secure API Gateway
+
+The ForgeRock Secure API Gateway for Open Banking is an open-source solution that helps financial services organizations deliver both test and production Open Banking and custom APIs from one framework, enforced by the ForgeRock Identity Platform.
+
+> These are the release notes for `Secure API Gateway` version `v1.0.1` named `Aerosmith`
```

Comment:
I think this would look better as a regular paragraph rather than a quote block

Reply:
Done!